pkg/pillar/hypervisor/kvm_test.go

```diff
@@ -90,7 +90,6 @@ func TestCreateDomConfigOnlyCom1(t *testing.T) {
 
```

Comment:
Can you please update kvm_test to test for rendering of USB assignments (bonus points -- payed in a liquid of your choice) is if you could also add PCI ones as well.

Reply:
Yes, let's add it to another PR